examples/compiled/layer_circle_independent_color.vg.json

```diff
@@ -212,7 +212,7 @@
       "title": "Acceleration (binned)",
       "labelFlush": true,
       "labelOverlap": true,
-      "tickCount": {"signal": "ceil(width/20)"},
+      "tickCount": {"signal": "ceil(width/10)"},
```

Comment:
I think this is just busier

<img width="907" alt="screen shot 2018-12-28 at 22 10 29" src="https://user-images.githubusercontent.com/589034/50534144-6df7e400-0aed-11e9-8193-4943fbd2ccb4.png">